Marlin/language_da.h

```diff
@@ -178,59 +186,55 @@
+#define MSG_FILAMENT_CHANGE_OPTION_HEADER   "Skift muligheder:"
 #define MSG_FILAMENT_CHANGE_OPTION_EXTRUDE  "Extruder mere"
-#define MSG_FILAMENT_CHANGE_OPTION_RESUME   "forsæt print"
-
-#if LCD_HEIGHT >= 4
-  #define MSG_FILAMENT_CHANGE_INIT_1          "Vent på"
-  #define MSG_FILAMENT_CHANGE_INIT_2          "skift af "
-  #define MSG_FILAMENT_CHANGE_INIT_3          "filament"
-  #define MSG_FILAMENT_CHANGE_UNLOAD_1        "Vent på"
-  #define MSG_FILAMENT_CHANGE_UNLOAD_2        "filament tømning"
-  #define MSG_FILAMENT_CHANGE_INSERT_1        "indsæt filament"
-  #define MSG_FILAMENT_CHANGE_INSERT_2        "og tryk for"
-  #define MSG_FILAMENT_CHANGE_INSERT_3        "at fortsætte..."
-  #define MSG_FILAMENT_CHANGE_LOAD_1          "Vent på"
-  #define MSG_FILAMENT_CHANGE_LOAD_2          "filament fødning"
-  #define MSG_FILAMENT_CHANGE_EXTRUDE_1       "Vent på"
-  #define MSG_FILAMENT_CHANGE_EXTRUDE_2       "filament extrudere"
-  #define MSG_FILAMENT_CHANGE_RESUME_1        "Vent på at"
-  #define MSG_FILAMENT_CHANGE_RESUME_2        "print forsætter"
-#else // LCD_HEIGHT < 4
-  #define MSG_FILAMENT_CHANGE_INIT_1          "Vent venligst..."
-  #define MSG_FILAMENT_CHANGE_UNLOAD_1        "Tømmer..."
-  #define MSG_FILAMENT_CHANGE_INSERT_1        "Indsæt og klik"
-  #define MSG_FILAMENT_CHANGE_LOAD_1          "Føder..."
-  #define MSG_FILAMENT_CHANGE_EXTRUDE_1       "Extrudere..."
-  #define MSG_FILAMENT_CHANGE_RESUME_1        "Fortsætter..."
-#endif // LCD_HEIGHT < 4
+#define MSG_FILAMENT_CHANGE_OPTION_RESUME   "Forsæt print"
+#define MSG_FILAMENT_CHANGE_INIT_1          "Vent på start"
+#define MSG_FILAMENT_CHANGE_INIT_2          "af filament"
+#define MSG_FILAMENT_CHANGE_INIT_3          "skift"
+#define MSG_FILAMENT_CHANGE_UNLOAD_1        "Vent på"
+#define MSG_FILAMENT_CHANGE_UNLOAD_2        "filament udskyd."
+#define MSG_FILAMENT_CHANGE_UNLOAD_3        ""
+#define MSG_FILAMENT_CHANGE_INSERT_1        "Indsæt filament"
+#define MSG_FILAMENT_CHANGE_INSERT_2        "og tryk på knap"
+#define MSG_FILAMENT_CHANGE_INSERT_3        "for at fortsætte..."
+#define MSG_FILAMENT_CHANGE_LOAD_1          "Vent på"
+#define MSG_FILAMENT_CHANGE_LOAD_2          "filament indtag"
+#define MSG_FILAMENT_CHANGE_LOAD_3          ""
+#define MSG_FILAMENT_CHANGE_EXTRUDE_1       "Vent på"
+#define MSG_FILAMENT_CHANGE_EXTRUDE_2       "filament extrudering"
+#define MSG_FILAMENT_CHANGE_EXTRUDE_3       ""
+#define MSG_FILAMENT_CHANGE_RESUME_1        "Vent på at print"
+#define MSG_FILAMENT_CHANGE_RESUME_2        "fortsætter"
+#define MSG_FILAMENT_CHANGE_RESUME_3        ""
+#define MSG_FILAMENT_CHANGE_INIT_1          "Vent venligst..."
+#define MSG_FILAMENT_CHANGE_UNLOAD_1        "Udskyder..."
+#define MSG_FILAMENT_CHANGE_INSERT_1        "Indsæt og klik"
+#define MSG_FILAMENT_CHANGE_LOAD_1          "Indtager..."
+#define MSG_FILAMENT_CHANGE_EXTRUDE_1       "Extrudere..."
+#define MSG_FILAMENT_CHANGE_RESUME_1        "Fortsætter..."
```

Comment:
As in the block above, these belong inside the conditionals. Please restore them as they were.